client/src/actions/team.js

```diff
@@ -218,18 +218,23 @@ export const getGHAccessToken = (teamId, code) => async (dispatch) => {
 };
 
 export const deployToGHPages =
-  (teamId, accessToken, twitterHandle) => async (dispatch) => {
+  (teamId, accessToken) => async (dispatch) => {
     try {
       // get publications
-      const { data } = await api.fetchPublicationsByTeamId(teamId);
-      data.map(
+      const { data: teamPublications } = await api.fetchPublicationsByTeamId(teamId);
+      teamPublications.map(
         (pub) => (pub.yearPublished = pub.yearPublished.substring(0, 4))
       );
+      //get teamInfo
+      const { data: teamInfo } = await api.getTeamJWT()
+      //get team members
+      const { data: teamMembers } = await api.fetchTeamMembersByTeamId(teamId);
 
       const body = {
         ghToken: accessToken,
-        teamTwitterHandle: twitterHandle,
-        teamPublications: data,
+        teamPublications,
+        teamInfo,
+        teamMembers
```

Comment:
Any specific reasons why we remove the keywords here?

Reply:
It's just a feature in ES6 that lets you get rid of the property name when the variable name is the same as the property name. Cause I find it looks cleaner in that way.  
https://ui.dev/shorthand-properties/